lib/Avatar/stories/AvatarGroupMock.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { number } from '@storybook/addon-knobs';
+import faker from 'faker';
+import uuid from 'uuid/v1';
+import AvatarGroup from '../AvatarGroup';
+import { AvatarWithPopover, ParticipantInfo } from '../../index';
+
+const AvatarGroupMock = ({ children, defaultValue }) => {
```

Comment:
In the work for #525 I added this function but now the item row story now uses it so I thought it would be a great idea to move this into a kind of mocking component.

Meaning whenever we want an AvatarGroup in stories you can use this mock rather than detailing our the same implementation every time.

Reply:
Love this 💟

Reply:
+1 great idea!